### ISSUE TITLE
Feature request-abilty to add a sound file to a key press

### ISSUE THREAD

**Issue by shaky1963** [DCS-Skunkworks/DCSFlightpanels#74]
Some DCS modules when you flick a switch in the cockpit you can hear a related sound file. Some modules such as KA50 no sound is heard when you flick a toggle switch. So a cool feature would be to add a sound file when programming a key press. So as well as having on the stream deck main page  "BUTTON ACTION" and "BUTTON IMAGE" you could have the option of "SOUND FILE".

**Comment by jdahlblom** [comment]
OK. Let's ask around if this is something others would use too. Do you have several sound files like this one?
[toggle_switch.zip](https://github.com/DCSFlightpanels/DCSFlightpanels/files/4640446/toggle_switch.zip)

**Comment by jdahlblom** [comment]
What if - I would embedd a few sound files with DCSFP? And user could choose from them? If embedded then there wouldn't be any need of keeping track of external files and sharing bindings with sound wouldn't be a problem.

**Comment by jdahlblom** [comment]
Or even simpler, just have an option of sound on / off. That would be the best.
Found some sound files here : https://freesound.org/search/?q=switch

**Comment by jdahlblom** [comment]
@shaky1963, check this pre-release => https://github.com/DCSFlightpanels/DCSFlightpanels/releases/tag/4.2.28
There you can specify a sound file for the button. Please provide feedback.

**Comment by shaky1963** [comment]
Thank you Guys
Will test tonight and provide feedback

**Comment by shaky1963** [comment]
Hi jdahlblom
I have done some initial testing, programming button presses with sounds in the KA50 and they work. The sounds i have used , i had downloaded from a free site. I have a request though - can you make the volume slider at maximum 100% louder? The key press sounds are hard to hear over the engine noise. My work around this at the moment is in the game settings - audio, i reduced the volume of all items by 50% and had dcsfp button push sound volume at 100%. Thank you for implementing this it is a good feature. Happy Easter and stay safe. Regards PEEP

**Comment by jdahlblom** [comment]
Thanks, same same!
I read about the volume control and it isn't possible. The value is between 0 - 1. The only option you have is to increase DCSFP's volume via Windows Volume Mixer which is located in the bottom right (task bar). I increased DCSFP's volume there and it worked. I am listening to Spotify at the same time so I had to lower Spotify's volume. (that too got increased??) 
Btw, I replaced the release with a new one, this new one plays the sound only when the key is pressed, not when it is released.
